arviz/plots/forestplot.py

```diff
@@ -88,39 +102,35 @@ def plot_forest(data, kind='forestplot', model_names=None, var_names=None, combi
         ncols += 1
         width_ratios.append(1)
 
-    plot_handler = PlotHandler(data, var_names=var_names, model_names=model_names,
-                               combined=combined, colors=colors)
+    plot_handler = PlotHandler(data, var_names=var_names, model_names=model_names, combined=combined, colors=colors)
 
     if figsize is None:
         figsize = (min(12, sum(width_ratios) * 2), plot_handler.fig_height())
 
-    (figsize, _, titlesize, xt_labelsize,
-     auto_linewidth, auto_markersize) = _scale_fig_size(figsize, textsize, 1.1, 1)
+    (figsize, _, titlesize, xt_labelsize, auto_linewidth, auto_markersize) = _scale_fig_size(figsize, textsize, 1.1, 1)
```

Comment:
@ahartikainen 120 line length works well on my desktop but is a bit of a stretch on my laptop. How strongly do you feel about 100 length vs 120?

Reply:
Not strong